### DISCUSSION TITLE
Default Branch Name Change

### DISCUSSION THREAD

**Discussion by srcarter3** [Unidata/python-awips#23] [General]
##   📢 **Quick Announcement:** 
We just wanted to let our users know, that we here at Unidata support inclusivity and that extends to language in our software.  With that in mind, we have recently renamed the **default** branch for python-awips from *master* to *main*.  If anyone has a local copy of the repo, please pull down or clone the new branch, or rename your local branch to be back in line with our github repo.  [Here is a blog post with detailed instructions if you need more help.](https://stevenmortimer.com/5-steps-to-change-github-default-branch-from-master-to-main/#:~:text=Steps%201%20Move%20the%20master%20branch%20to%20%E2%80%98main%E2%80%99,5%20Delete%20%E2%80%98master%E2%80%99%20branch%20on%20the%20remote%20repo) Thanks!

**Comment by srcarter3** [comment]
TLDR:
Update your local branch with these lines (assuming your repo name is 'origin'):

```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```
